app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    split = command.split(" ")
+    command = split[0]
+
+    if command != "cp":
+        return
+
+    file1 = split[1]
+    file2 = split[2]
+
+    if file1 == file2:
+        return
+
+    with open(file1, "r") as file_in, open(file2, "w") as file_out:
```

Comment:
Consider adding error handling for file operations to manage cases where files might not exist or there are permission issues.